src/main/java/lotto/domain/Amount.java

```diff
@@ -23,12 +23,20 @@ public void add(Long value) {
         this.amount += value;
     }
 
+    public int purchasableLottoCount() {
```

Comment:
`Amount` 입장에서는 `Lotto`는 어색합니다.

```suggestion
    public int divide(int 나눌값) {
```